_meetups/2023-05-11-eagerworks.md

```diff
@@ -12,6 +12,7 @@ talks:
      url: https://github.com/ignaciogrondona
 
  - title: Developer Experience in Ruby
+   slides: https://speakerdeck.com/maicolben/developer-experience-in-ruby
```

Comment:
You can also point to the embedded player.

```suggestion
   slides: https://speakerdeck.com/player/b98a3c0bc61247b28ca3c462c09d190f
```

I'm going to add that to the README

Reply:
thanks!